tests/Parsers/ItemParserTest.php

```diff
@@ -645,6 +645,25 @@ public function itParsesMeta()
         static::assertEquals(new Meta(['foo' => 'bar']), $item->getMeta());
     }
 
+    /**
+     * @test
+     */
+    public function itParsesMetaInData()
+    {
+        $typeMapper = new TypeMapper();
+        $typeMapper->setMapping('child', ChildItem::class);
+        $typeMapper->setMapping('master', MasterItem::class);
+        $parser = $this->getItemParser($typeMapper);
+
+        $item = $parser->parse($this->getJsonApiItemMock('master', '1'));
+
+        $dataMeta = $item->getRelation('childwithdatameta')->getIncluded()->getDataMeta();
+        static::assertInstanceOf(Meta::class, $dataMeta);
+
+        $image = $dataMeta->imageDerivatives->links->header->href;
+        $this->assertEquals('https://example.com/image/header/about-us.jpeg', $image);
```

Comment:
This is so we can check we're getting specific information back, and not accidentally returning the other Meta object.